include/swift/AST/Import.h

```diff
@@ -108,6 +109,14 @@ namespace detail {
       if (empty()) return SourceRange();
       return SourceRange(raw.front().Loc, raw.back().Loc);
     }
+
+    StringRef str() const {
+      SmallString<64> modulePathStr;
+      llvm::interleave(
+          *this, [&](Element elem) { modulePathStr += elem.Item.str(); },
+          [&] { modulePathStr += "."; });
+      return modulePathStr.str();
```

Comment:
The return of a value derived from a temporary here will cause a use-of-stack-after-return.